src/app/fyle/add-edit-advance-request/add-edit-advance-request.page.ts

```
@@ -473,7 +473,7 @@ export class AddEditAdvanceRequestPage implements OnInit {
       componentProps: {
         header: 'Delete Advance Request',
         body: 'Are you sure you want to delete this request?',
-        deleteMethod: (): Observable<AdvanceRequests> =>
+        deleteMethod: (): Observable<{}> =>
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)